exercises/basiccalcobject.js

```diff
@@ -1,18 +1,44 @@
 
 class Calculator{
 	constructor(){
```

Comment:
One of the most important aspects of the constructor in a class constructor is that it serves as the global space for the object. In this example of a calculator object, the constructor is acting as a storage space for the operator that will eventually be passed in, as well as the numbers that will be part of the calculation. The constructor is incredibly important in this way, because it allows you to store information in an object that is readily available to all the methods attached to that object. This allows you to do two things, organize the data the object methods need to function, and to centralize data related to the object (for example the “name” of a person object, or the numbers/operator data present in this example). Organization is extremely important in programming, as I am sure you have noticed, and constructors are very helpful in this regard.